services/notifications/pkg/email/templates.go

```diff
@@ -7,7 +7,8 @@ func Template(s string) string { return s }
 var (
 	// Shares
 	ShareCreated = MessageTemplate{
-		bodyTemplate: "shares/shareCreated.email.body.tmpl",
+		textTemplate: "shares/shareCreated.email.body.tmpl",
```

Comment:
Can we also have only one template for text emails?

Reply:
Yes, we can

Reply:
It could be `textTemplate: "text/email.text.tmpl",`

Reply:
Already changed to ```textTemplate: "text/email.text.tmpl"```